kore/src/SMT.hs

```diff
@@ -208,6 +210,11 @@ check :: MonadSMT m => m (Maybe Result)
 check = liftSMT checkSMT
 {-# INLINE check #-}
 
+-- | Check if the current set of assertions is satisfiable.
```

Comment:
I think this should read as something like "get values from the model of a satisfiable set of assertions", since it does not check satisfiability, but rather assumes it has already been checked.